zutty.tera

```diff
@@ -0,0 +1,38 @@
+---
+whiskers:
+  version: 2.0.0
+  matrix:
+    - flavor
+  filename: "{{flavor.identifier}}.Xresources"
```

Comment:
perhaps place the themes in a `theme` dir like some of the other ports.

Reply:
hmm fair enough